src/components/RollupProductDevDoc.js

```diff
@@ -0,0 +1,64 @@
+// Libraries
+import React from "react"
+import styled from "styled-components"
+
+// Components
+import Link from "./Link"
+
+// Data
+import optimisticRollupList from "../data/layer-2-data-files/optimistic-rollups.json"
+import zkRollupList from "../data/layer-2-data-files/zk-rollups.json"
+
+// Styles
+const H3 = styled.h3`
+  margin-top: 0;
+  margin-bottom: 1rem;
+`
+
+const ProductCard = styled.div`
+  margin: 1rem 0;
+  background: ${(props) => props.theme.colors.searchBackground};
+  display: flex;
+  flex-direction: row;
+`
+
+const Content = styled.div`
+  padding: 1rem;
+`
+
+const RollupProductDevDoc = ({ rollupType }) => {
+  const data = {
+    optimistic: optimisticRollupList,
+    zk: zkRollupList,
+  }
+
+  return (
+    <div>
+      {data[rollupType].map((rollup) => {
+        return (
+          <ProductCard>
+            <Content>
+              <div>
+                <H3>
+                  {rollup.name}
+                  {rollup.note.length > 0 && ` (${rollup.note})`}
+                </H3>
```

Comment:
I think this should be an h4 as it's a child of an h3.

<img width="724" alt="Screenshot 2022-02-28 at 12 35 23" src="https://user-images.githubusercontent.com/62268199/155984499-1c29b955-be33-4ca6-aa76-5b8684246738.png">


```suggestion
                <H4>
                  {rollup.name}
                  {rollup.note.length > 0 && ` (${rollup.note})`}
                </H4>
```